lib/autokey/gtkui/popupmenu.py

```diff
@@ -28,13 +27,17 @@ class PopupMenu(Gtk.Menu):
     A popup menu that allows the user to select a phrase.
     """
 
-    def __init__(self, service, folders=[], items=[], onDesktop=True, title=None):
```

Comment:
This is a subtle change.
I hope that the code does not rely on implicit sharing of a single list instance across calls.